litex/boards/targets/arty_symbiflow.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
```

Comment:
Why do we need this target?

Reply:
Just for testing purposes. Original Arty target doesn't work, because Symbiflow doesn't support all Xilinx primitives (e.g. IOBUFDS) yet. It also requires buffers on input clock.